diff --git a/lifted-async.cabal b/lifted-async.cabal
index abe432d..4af96c8 100644
--- a/lifted-async.cabal
+++ b/lifted-async.cabal
@@ -32,7 +32,7 @@ library
     Control.Concurrent.Async.Lifted
     Control.Concurrent.Async.Lifted.Safe
   build-depends:
-      base >= 4.5 && < 4.14
+      base >= 4.5 && < 4.15
     , async >= 2.2 && < 2.3
     , lifted-base >= 0.2 && < 0.3
     , transformers-base >= 0.4 && < 0.5
