diff --git a/time-compat.cabal b/time-compat.cabal
index 1681341..5493c41 100644
--- a/time-compat.cabal
+++ b/time-compat.cabal
@@ -115,7 +115,7 @@ test-suite main
   ghc-options:        -Wall -fwarn-tabs
   build-depends:
       base
-    , base-compat       >=0.10.5 && <0.11
+    , base-compat       >=0.10.5 && <0.12
     , deepseq
     , QuickCheck        >=2.13   && <2.14
     , tagged            >=0.8.6  && <0.9
